src/main/resources/fetch.sql

```diff
@@ -0,0 +1,18 @@
+SELECT * FROM homework;
```

Comment:
Please never use `*` to select everything from the table. Would be much better to specify all columns you need by their names. JFYI, if you do `select * from table` during the job interview - this will badly impact the result.